learning/dashboard/BuildingBlock/README.md

```diff
@@ -0,0 +1,459 @@
+
+
+
+## Benefits of CSS3
+
+ - Using CSS3, navigable components can be positioned consistently and
+   precisely.
+ - A web page can be easily customized because it only requires making  
+   minor changes to a modular file.
+ - CSS3 makes it simpler to create graphics, which makes it simple to   
+   make the site appealing. It enables the viewing of online videos   
+   without the need for external plug-ins.
+ - CSS3 is affordable, quick, and   
+   supported by the majority of browsers.
+## CSS3 EXAMPLES
+ - ### Web Font : 
+  A Web Font is just a technique to use any font in your page, being downloaded instantly by the browser.This will be a radical departure from web design's prior use of just 10 to 15 generally accepted fonts. Yet, because only typefaces with valid licences should be used, this new function raises questions about copyright.
+
+
+    @font-face  {
+    font-family:  'Name of the new font';
+    src:  url('https://designshack.net/fonts/font.ttf');
+    }
+![Example of page have been mocked up using this technique.](https://designshack.net/postimages/text_webfonts1.jpg)
+
+
+
+ - ### USER INTERFACE :
+ #### Resize 
+
+With CSS3, you can quickly apply resizing to any element to make it cross-browser compatible.
+
+    .ui_resizable  {
+    padding:  20px;
+    border:  1px  solid;
+    resize:  both;
+    overflow:  auto;
+    }
+
+
+
+#### Multiple Columns Using CSS3
+Although CSS2 already has the ability to set an element's outline, CSS3 adds the ability to offset the outline from the element by a value you specify. It varies from a boundary in two ways:
+
+Outlines don't occupy any columns.
+Not all outlines must be rectangular.
+Multi-column layout is a new module added to CSS3 that is suitably named. It lets you to select how many columns text should be divided down into and how they should appear.
+
+    .multiplecolumns  {
+    -moz-column-width:  130px;;
+    -webkit-column-width:  130px;
+    -moz-column-gap:  20px;
+    -webkit-column-gap:  20px;
+    -moz-column-rule:  1px  solid  #ddccb5;
+    -webkit-column-rule:  1px  solid  #ddccb5;
+
+   ![enter image description here](https://designshack.net/postimages/multiplecolumns.png)
+
+
+### Spanning Columns
+
+span more than one column – a heading, table or image for instance.
+
+    enter code hereh2  {
+    column-span:  all
+    }
+![enter image description here](https://designshack.net/postimages/multiplecolumns_heading.jpg)
+  
+  #  INTRODUCTION TO LSS
+- Less (Leaner Style Sheets) is a dynamic preprocessor style sheet language that can be compiled into Cascading Style Sheets (CSS) and run on the client side or server side.
+-   It was Designed by Alexis Sellier.
+
+- ESS is an abbreviated `Learner Style styles`, is an opensource pre-processor language for CSS, The preprocessor is writing a code using a superset of CSS features, compiling this code to native CSS, and browser understanding CSS styles.
```

Comment:
What is ESS? are you referring to LESS?
Looks like the content here is repeated. May want to check and keep only relevant content.